lib/src/HttpUtils.h

```diff
@@ -18,14 +18,16 @@
 #include <drogon/HttpTypes.h>
 #include <string>
 #include <trantor/utils/MsgBuffer.h>
+// To export symbol to unit tests
+#include <drogon/exports.h>
 
 namespace drogon
 {
 const string_view &webContentTypeToString(ContentType contenttype);
 const string_view &statusCodeToString(int code);
 ContentType getContentType(const std::string &fileName);
-ContentType parseContentType(const string_view &contentType);
-FileType parseFileType(const string_view &fileExtension);
+DROGON_EXPORT ContentType parseContentType(const string_view &contentType);
```

Comment:
I'm not sure this is the best idea. I understand that theses functions are not meant to be accessible from the outside. But MSVC can't see the symbols leading to the unittest failing to build.

Reply:
Could we EXPORT the symbols in the test code?

Reply:
No. MSVC decides to export or not when compiling the function implementations (i.e. libdrogon.dll).

Reply:
The other way around this would be to add HttpUtils.cc as a source file of the unittest if and only if we are on MSVC and building DLLs. I'm not sure which is less invasive.

Reply:
I prefer the last way.

Reply:
Should we install the drogon_test.h to the destination? I think it is helpful for tests of applications.  @marty1885

Reply:
Sure, I don't see why it can't be used that way.